OpenSim/Common/Component.h

```diff
@@ -627,6 +627,12 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
      * and, therefore, calling this method is not "free". */
     std::string getAbsolutePathName() const;
 
+    /** Return a ComponentPath of the absolute path of this Component. 
+     * Note that this has more overhead than calling `getName()` because 
+     * it traverses up the tree to generate the absolute pathname (and 
+     * is thus a function of depth). Consider other options if this is
```

Comment:
`and is thus` -> `and its computational cost is thus`

Reply:
I forgot that the merge didn't change the comment I had written. An alternative is to use the same comment from the function above (`The absolutePath is generated ... not "free."`).